crates/biome_formatter/src/lib.rs

```diff
@@ -292,8 +339,28 @@ impl<'de> serde::Deserialize<'de> for LineWidth {
 pub enum ParseLineWidthError {
```

Comment:
The name of the type seems no longer consistent with the possible errors.

Reply:
Can you suggest an alternative?